content/en/installing-pixie/03-install-guides/02-self-hosted-pixie.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Quick Start"
-metaTitle: "Install | Quick Start"
-metaDescription: "Getting started guide to setup Pixie"
+title: "Self-Hosted Pixie"
+metaTitle: "Install | Install Guides | Self-Hosted Pixie"
+metaDescription: "Getting started guide to setup 100% self-hosted Pixie"
 order: 2
```

Comment:
Let's shorten the "2. Install the Pixie CLI" section as done in the Community Cloud Guide.